assets/javascripts/openqa.js

```diff
@@ -534,3 +534,15 @@ function renderComments(row) {
   }
   return html;
 }
+
+function renderHttpUrlAsLink(value) {
+  const text = document.createTextNode(value);
+  if (!value.match(/^https?:\/\//)) {
+    return text;
+  }
+  const link = document.createElement('a');
+  link.href = value;
+  link.target = 'blank';
+  link.appendChild(text);
+  return link;
+}
```

Comment:
No, it is no duplicate. The job details settings are rendered in Perl code and this is rendered client-side. Apart from the fact that client-side rendering is the modern approach, it also needs to be client-side because the rendering is needed for a modal dialog we show dynamically when a button in a DataTables row is clicked. I also checked whether we already have such a helper in our existing JavaScript code.